src/object/schema_manager.c

```diff
@@ -2168,6 +2169,89 @@ sm_downcase_name (const char *name, char *buf, int maxlen)
   intl_identifier_lower (name, buf);
 }
 
+/* AUTO_INCREMENT_SERIAL_NAME_MAX_LENGTH is not a target. 
+ * Trigger and serial names can be longer than DB_MAX_IDENTIFIER_LENGTH.
+ */
+int
+sm_user_specified_name (const char *name, const char *owner_name, char *buf, int buf_size)
```

Comment:
In all cases where this function is called, owner_name is specified as NULL. 
If it is unnecessary, it is better to organize it.

Reply:
Thanks to you, I realized I was calling owner_name all NULL.

So I changed the sm_user_specified_name function.
1. It does not receive owner_name.
2. When name is not user_specified_name, current_user_name is used to make it user_specified_name.
3. The sm_user_specified_name function is called in sm_find_class.
4. Calls from other unnecessary locations have been removed.